pubsub.go

```diff
@@ -25,17 +26,24 @@ import (
 
 var log = logging.Logger("pubsub-valuestore")
 
+const PSGetLatestProto = protocol.ID("/pubsub-get-latest/0.0.1")
```

Comment:
let's pick a better protocol name; also, prefix it with `/libp2p` to scope it as a canonical libp2p protocol implementation.

Reply:
Any suggestions? I don't have a strong preference, @Stebalien suggested the above.

Reply:
how's `/libp2p/fetch/0.0.1`?

The protocol has no explicit ties to pubsub or anything being the "latest", it's basically just the function `Fetch(key string) []byte` and it may actually make sense to change it to `Fetch(key []byte)  []byte` as well.

Reply:
it's better, but still not perfect.

Reply:
Ok. Should we change it to `Fetch(key []byte) []byte`?

Reply:
> it's better, but still not perfect.

Any better suggestions? I don't want this blocked on a naming issue.

Reply:
Something like `/libp2p/namesys/fetch`?

Reply:
Not sure why bother to use `namesys` here since this could be used for plenty of non-namesys things. For example, `key` could be a `CID` then it does content fetching.